packages/shared/forks/ReactFeatureFlags.test-renderer.js

```diff
@@ -61,7 +61,7 @@ export const enableSyncDefaultUpdates = true;
 export const allowConcurrentByDefault = false;
 export const enablePersistentOffscreenHostContainer = false;
 
-export const consoleManagedByDevToolsDuringStrictMode = true;
+export const consoleManagedByDevToolsDuringStrictMode = false;
```

Comment:
I'd expect this to be false because test renderer doesn't talk to DevTools

Reply:
This (and other test flag changes below) seems reasonable.